### ISSUE TITLE
Setup CI for Self Deposit application

### ISSUE THREAD

**Issue by eporter23** [emory-libraries/dlp-selfdeposit#123]
This ticket is a placeholder to followup on Slack discussions regarding the need to set up CI for the new application.

Proposal: use GitHub Actions (to align with Samvera practices) instead of Circle CI
Note: may have some differences because we have GitHub enterprise. 
If we run into issues, we may have to revert back to using Circle CI

**Comment by abelemlih** [comment]
I ran into a major blocker setting up GitHub Actions: the application's Solr setup requires the use and configuration of both Solr and Zookeeper, and I have not found a reliable path to configuring both on GitHub Actions. These tasks are typically fulfilled on CircleCI through the samvera/circleci-orb: https://circleci.com/developer/orbs/orb/samvera/circleci-orb (refer to task `install_solr_cor`).

I talked to a member of the Samvera community who offered a few options currently considered to resolve this issue. I will discuss those options with Collin tomorrow to determine the best step forward.
